src/graphql/resolvers/postOfferResolver.js

```diff
@@ -33,17 +33,17 @@ const PostOfferResolver = async (db, args) => {
   };
   let companyName = null;
   let company = await db.query(companyParams).promise();
-  let uuid = uuidv4();
-  // companyName = company.Items[0].name;
-  console.log(company);
+  let uuid;
   if (company.Items.length === 0) {
```

Comment:
Do we have a task to refactor this method? Once the tests are in case, it should be pretty simple to refactor.

Reply:
Following up on this ^